src/mixins/localize-mixin.js

```diff
@@ -17,12 +17,19 @@ export default dedupingMixin(
            * string should be passed in order, as follows:
            * `localize(stringKey, param1Name, param1Value, param2Name, param2Value)`
            */
+          language: {
```

Comment:
The comment above is for the prop below.

Also add a comment why we have added language again.

Reply:
I wonder if the correct fix is to just to find & replace `this.language` with `this.hass.language`

Reply:
That also solves the problem... Occurs twice in ha-state-history-data.js

Reply:
Also makes more sense